docs/intro/ff-ui/dashboard.md

```diff
@@ -0,0 +1,81 @@
+
+![dashboard](imgs/dashboard.avif)
+
+- **Projects**: This section displays the projects you have created in FlutterFlow. You can easily navigate to any of them from here. Use the overflow menu to rename, duplicate, delete, leave the project, add tags, and open the project in a new browser tab.
+
+    :::info
+    When you duplicate a project containing the Firebase setup, you must delete the config files in your duplicated project and initiate a new [**Firebase setup**](../../ff-integrations/firebase/connect-to-firebase-setup.md) for your project.
+    :::
+
+- **Dark/Light mode switch**: The Dark/Light mode switch allows you to choose between a light and dark color scheme for our app builder.
+
+- **Search**: From here, you can search for your project.
+
+- **Filter projects**: You can use this dropdown menu to filter your projects based on their privacy settings. You can filter projects that are marked as private, shared by you, or shared with you.
+
+- **Tag project**: From here, you can create and add a tag to projects, providing a quick and organized way to classify and identify projects based on their characteristics, purpose, or status.
+
+    To tag a project, simply click the three dots option menu on the project tile, select **Add Tag,** and then click on the tag you want to set. Later, you can search and filter the projects based on the tags.
+
+    <div style={{
+        position: 'relative',
+        paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+        height: 0,
+        width: '100%'}}>
+        <iframe 
+            src="https://www.loom.com/embed/2e545489cfe14e97ae44b6a747410ae0?sid=b9a93abe-7fb7-4e16-a014-6477543c4460"
+            title=""
+            style={{
+                position: 'absolute',
+                top: 0,
+                left: 0,
+                width: '100%',
+                height: '100%',
+                colorScheme: 'light'
+            }}
+            frameborder="0"
+            loading="lazy"
+            webkitAllowFullScreen
+            mozAllowFullScreen
+            allowFullScreen
+            allow="clipboard-write">
+        </iframe>
+    </div>
+    <p></p>
+
+- **Create a new project**: For creating a new project, you can use the **+** **Create New** button. Learn more about [creating a new project](../../resources/projects/how-to-create-find-organize-projects.md#how-to-create-a-project).
+
+- **Notifications Center**: The notification center simplify how you manage comments and invites across various projects. It conveniently centralizes all your project communications. And when you're ready to address a comment, simply tap on it, and you'll be taken directly to the relevant section of the project.
+
+- **My organization** From [here](my-organization.md), you can share custom code, assets, design systems, and APIs between your team users and across projects.
```

Comment:
Let's avoid saying "here" as much as possible. 1. not good for search indexing, and 2. can easily be replaced by the name for better context. This applies to callout links too. 
E.g
avoid: "To know more about widgets, see here"
instead: "Learn more about widgets and widget trees." And add links to the main words itself. 

I'll add this to style guide too. See this [commit](https://github.com/FlutterFlow/flutterflow-documentation/pull/118/commits/c2b4fdc766fa243ed68c23901afbb02239b9d165) to see how we can approach such sections.